docs/warehouse/warehouse_starter_kit.md

```diff
@@ -0,0 +1,58 @@
+    * Use `gtfs_utils_v2.get_trips()`.
+    * Answer how many trips is an operator scheduled to run? How many trips did a particular route make?
+
+* [fct_observed_trips](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_observed_trips)
+    * Realtime observations of trips combined with scheduled data to get a full picture of what occurred.
+    * Find a trip's start time, where it went, and which route it is associated with.
+
+## Shapes
+* [fct_daily_scheduled_shapes](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_shapes)
+    * Use `gtfs_utils_v2.get_shapes()`.
+    * Contains `point` geometry, so you can see the length and location of a route an operator can run on a given date.
+    * Find out how many trips did an operator make for a particular route.
+    * Each shape has its own `shape_id` and `array_key`.
+    * An express version and the regular version of a route are considered two different shapes.
+
+## Daily
+For a given day:
+* [fct_daily_scheduled_stops](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_scheduled_stops)
+    * Use `gtfs_utils_v2.get_stops()`.
+    * Contains `point` geometry.
+    * How many stops did an operator make? Where did they stop?
+    * How many stops did a particular transit type (streetcar, rail, ferry...)?
+    * Detailed information such as how passengers embark/disembark (ex: on a stop/at a station) onto a vehicle.
+
+* [fct_daily_schedule_feeds](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_schedule_feeds)
+    * Use `gtfs_utils_v2.schedule_daily_feed_to_organization()`.
+    * Get information on an operator's schedule.
+    * How many different schedule types (schedule, vehicle positions, trip updates, and/or service alerts) does an operator produce?
+    * Is the URL to an operator's GTFS feed stable or not?
+
+### Other
+* [dim_annual_database_agency_information](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.dim_annual_database_agency_information)
+    * View some of the data produced by the [US Department of Transportation](https://www.transit.dot.gov/ntd) for the National Transit Database.
+    * Information from 2018-2021 are available.
+    * Includes information such as reporter type, organization type, website, and address.
+    * Not every operator is required to report their data to the NTD, so this is not a comprehensive dataset.
+
+* [fct_daily_organization_combined_guideline_checks](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_organization_combined_guideline_checks)
+    * Understand GTFS quality - how well an operator's GTFS data conforms to [California's Transit Data Guidelines](https://dot.ca.gov/cal-itp/california-transit-data-guidelines).
+    * Each operator has 14 rows. Each row details how well an operator's GTFS data conforms to a certain guideline (availability on website, accurate accessibility data, etc).
```

Comment:
As of today it looks like there are 74 rows/day per organization

Reply:
Each operator has 74 rows per day?

Reply:
Just to double check: There may be a bit of confusion about language here -- in the Airtable context, "operator" means the organization that physically operates the vehicles for the service (which may sometimes be a private contractor), and "provider" is the organization that manages the service (the organization we would often refer to as "the agency" or perhaps the agency's parent organization, something like a city or county government). This table contains information about transit providers (rather than "operators") in the Airtable sense of the word. 

That being said, yes, there are 74 rows per organization (in this case, an assessed transit provider) per day in this table at present. That number could continue to grow a little. One row in this table represents one guideline check for a given organization. There are 74 rows per day because there are 74 separate guideline checks being performed. The list of implemented checks [is available here](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.int_gtfs_quality__guideline_checks_long).

I am also happy to chat about this offline if helpful! I realize it's confusing and I don't mean to be overly pedantic with the language, just want to make sure I'm not accidentally using words in a different way 😅

Reply:
(Also perhaps worth noting: This row count can change if we implement additional guideline checks, so perhaps it will be easier to maintain if the row count is not listed in the docs since it may be difficult to maintain.)